service.js

```diff
@@ -56,6 +55,17 @@ seneca.use(require('seneca-queue'));
 seneca.use(require('cp-permissions-plugin'), {
   config: `${__dirname}/config/permissions-rules`
 });
+if (process.env.NEW_RELIC_ENABLED === 'true' ) {
+  seneca.use(require('seneca-newrelic'), {
+    newrelic,
+    roles: ['cd-dojos'],
+    filter: (p) => { 
+      p.user = p.user ? p.user.id : void 0;
+      p.login = p.login ? p.login.id : void 0;
```

Comment:
kind of personal/petty but `undefined`  is more readable/understandable then `void 0`

Reply:
Same. `void 0` is weird to me, but meh